paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py

```diff
@@ -115,19 +115,19 @@ def download_unzip():
 
     data_dir = os.path.join(target_folder, 'ILSVRC2012')
     print('ILSVRC2012 full val set at {0}\n'.format(data_dir))
-    return data_dir
+    return cache_folder, data_dir
 
 
 def reader():
-    data_dir = download_unzip()
-    file_list = os.path.join(data_dir, 'val_list.txt')
-    output_file = os.path.join(data_dir, 'int8_full_val.bin')
+    des_dir, source_dir = download_unzip()
+    file_list = os.path.join(source_dir, 'val_list.txt')
+    output_file = os.path.join(des_dir, 'int8_full_val.bin')
     with open(file_list) as flist:
         lines = [line.strip() for line in flist]
         num_images = len(lines)
         if not os.path.exists(output_file):
             print(
-                'Preprocessing to binary file...<num_images><all images><all labels>...\n'
+                'Start to convert jpg to binary file, the file will be in form [num_images | all images in binary | all labels in binary]...\n'
```

Comment:
The log info is too long and too confused. Maybe you can change it to "Start to convert jpg to binary file, the file will contain images and labels data"

Reply:
OK.

Reply:
Please add "Start download Full ImageNet Validation dataset" before starting download the dataset, remove the print on line 109 and line 114.